src/Google/Service/Resource.php

```diff
@@ -232,6 +200,37 @@ public function call($name, $arguments, $expectedClass = null)
     return $this->client->execute($request, $expectedClass);
   }
 
+  private function logServiceError($txt, $name, $parameter = null)
+  {
+    $array = array(
+        'service' => $this->serviceName,
+        'resource' => $this->resourceName,
+        'method' => $name,
+        'parameter' => $parameter
+    );
+    if ($parameter) {
+        $array['parameter'] = $parameter;
+    }
+    $this->client->getLogger()->error(
+        $txt,
+        $array
+    );
+  }
+  
+  private function logServiceInfo($txt, $name, $parameters)
+  {
+    $array  = array(
```

Comment:
Let's use a variable name like `$context`, to match with the [PSR spec](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md#3-psrlogloggerinterface).